packages/ai-chat-ui/src/browser/chat-input-widget.tsx

```diff
@@ -65,9 +67,12 @@ export class AIChatInputWidget extends ReactWidget {
         this.title.closable = false;
         this.update();
     }
+
     protected override onActivateRequest(msg: Message): void {
         super.onActivateRequest(msg);
-        this.node.focus({ preventScroll: true });
+        if (this.editorRef) {
+            this.editorRef.focus();
+        }
```

Comment:
On first rendering of the view, the `editorRef` does not exist yet. So the input will not get focus. I think we need to work with a Promise which resolves once the editor is available.

Reply:
Hm, I can't reproduce the issue. If I have the view closed, refresh and then open it, the focus is correctly obtained.
If I have the view open and then refresh, the input receives focus, but then something else steals it in Theia, which I guess is fine, I don't know if there is a dedicated focus priority.

Reply:
![InitialOpeningChat](https://github.com/user-attachments/assets/faa5927d-c384-404d-aba8-c83446bc1635)

Reply:
Thanks for demonstrating the case! This is quite curious, because for me this works on a clean build of the exact branch commit:

https://github.com/user-attachments/assets/77f71eee-81e4-4735-b582-0da0798de5a9

Anyway, I understand why this may not work and will try to propose a fix.

Reply:
I added a new commit ([1862f9b](https://github.com/eclipse-theia/theia/pull/14608/commits/1862f9ba4434c8c8d7c207d90c6ff727fc88798e)), can you please check if this now works for you? Thank you!